src/lazy/binary/raw/v1_1/value.rs

```diff
@@ -360,6 +361,33 @@ impl<'top> LazyRawBinaryValue_1_1<'top> {
         }
     }
 
+    pub(crate) fn for_fixed_uint8(input: BinaryBuffer<'top>) -> Self {
```

Comment:
Need to update this to support other uint sizes. It's not a problem currently since there's only 1 size implemented in this PR, but it will be updated to support the other size uints.